### PR TITLE
Fix expressions; Improve guide

### LINKED ISSUE #106: 8vim's github description has errors.
https://github.com/flide/8VIM/

"_The application was started essentially as a clone of 8pen, which for some reason they have stopped distributing. (I can no longer install it even though when I paid for it!!!)_"

`even though when` is incorrect. I think it would be correct if `when` was deleted.

"_The application at first was created as a clone of 8pen, which for some reason they have stopped distributing. (I can no longer install it even though I paid for it!!!)_"

### PR DESCRIPTION
**Pull request by m-rey**
I've made some changes to the readme to better explain the functionality of the keyboard. I've also improved spelling and grammar in general.
This should also fix #106.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,47 +18,47 @@ To understand what exactly this keyboard is, please watch the video "https://www
...
+##### Basic Stuff
 
-- Right Sector acts as a backspace Key.
-- Bottom Sector acts as an Enter Key.
-- Top Sector acts as a combo of SHIFT and CAPS_LOCK Key, i.e, Press once Shift is active, Press Twice CAPS is active and Press once more and everything is back to usual.
-- Left Sector acts as the button that takes you to the Number Pad.
+- right sector acts as a Backspace key.
+- bottom sector acts as an Enter Key.
+- top sector acts as a combo of Shift and Caps Lock key (i.e., press once to activate Shift. Press again for Caps Lock. Press once more and everything is back to normal.)
+- Left Sector acts as the button that takes you to the number pad.
```
Comment (Glitchy-Tozier) [inline README.md]: If the bullet-points above aren't capitalized, i think the last one one shouldn't be either.

**Comment by Glitchy-Tozier** [comment]
Amazing work, @m-rey!
Also, I inserted a comment in your changes ... did it show up? `[edit: apparently yes, it showed up]`

File: README.md (hunk 1 above)
Comment (m-rey) [reply README.md]: Good catch! Fixed it

**Comment by m-rey** [comment]
I've added some further corrections that I apparently missed last time. @Glitchy-Tozier feel free to review again :)

**Comment by Glitchy-Tozier** [comment]
Great work, I see no problems with it  :)

**Review by flide — APPROVED** [review]

**Pull request merged**